examples/binding-patterns-in-match-statement/binding_patterns_in_match_statement.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/io;
+
+type Pair record {
+    int x;
+    int y;
+};
+
+function foo(Pair pair) {
+    match pair {
+        // Below binding pattern also has a rest binding pattern to capture the additional fields
+        // that may be defined in pair, since it is an open record.
+        // Type of rest can be considered a map of anydata, however it cannot contain the keys
+        // x or y. This is achieved using never type;
```

Comment:
Type of rest is explore more in the `never` section, should it not be mentioned here?

Reply:
Each BBE is expected to make sense on its own. 

Also when we say "This **is** achieved using never type", I personally would expect an example of it here.

Maybe we can say "This can be represented using the `never` type as explained in the example for the `never` type."